pom.xml

```diff
@@ -228,6 +228,13 @@
             <artifactId>velocity-tools</artifactId>
             <version>1.4</version>
         </dependency>
+
+        <dependency>
+            <groupId>oro</groupId>
+            <artifactId>oro</artifactId>
+            <version>2.0.8</version>
+        </dependency>
```

Comment:
Interesting. You should not be able to pass the tests without oro. EscapeReference the parent of EscapeHtmlReference uses oro. Can you provide a simple reproducer?